### PR TITLE
Cartographer: Added note for ardupilot_ros

### PR DESCRIPTION
**Pull request by snktshrma**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
shouldnt we somehow move this into the AP repo structure? linking to externals is not the best... @peterbarker ?

**Comment by rmackay9** [comment]
Yes, we've now forked the repo into AP so we can update the note to point here https://github.com/ArduPilot/ardupilot_ros

**Review by rmackay9 — APPROVED** [review]

File: dev/source/docs/ros-cartographer-slam.rst [hunk 1]
```diff
@@ -236,6 +236,10 @@ After <rosparam command="load" file="$(arg config_yaml)" /> add a line like belo
 ::
 
     <remap from="/mavros/vision_pose/pose" to="/robot_pose" />
+    
+.. note::
+
+    Instead of maually modifying the above packages, clone `this <https://github.com/ArduPilot/ardupilot_ros>`__ repository and install the dependencies.
```
Comment (rmackay9) [inline dev/source/docs/ros-cartographer-slam.rst:242]: "maually" should be "manually"

**Comment by rmackay9** [comment]
Great, let's just squash these commits together and then I think we are done.

**Comment by snktshrma** [comment]
Squashed the commits @rmackay9 sir

**Comment by rmackay9** [comment]
Thanks, looks good to me.

**Pull request merged**